spec/mailers/notifier_spec.rb

```diff
@@ -318,6 +318,23 @@
     end
   end
 
+  context "limited comments" do
+    let(:commented_limited_post) {bob.post(:status_message, :text => "### Limited headline \r\n It's **really** sunny outside today, and this is a super long status message!  #notreally", :to => :all, :public => false)}
+    let(:limited_comment) { eve.comment!(commented_limited_post, "Totally is")}
+
+    describe ".comment_on_limited_post" do
+      let(:limited_comment_mail) {Notifier.comment_on_post(bob.id, person.id, limited_comment.id).deliver_now}
+
+      it 'SUBJECT: does not show limited message' do
+        expect(limited_comment_mail.subject).not_to include("Limited headline")
+      end
+
+      it 'BODY: does not show limited message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.